docs/IonCorePublishing.md

```diff
@@ -0,0 +1,38 @@
+# Ion Core Extension Publishing HOWTO
+
+## First-time
+
+Since the core extension requires special Firefox-only privileges, it must use the Mozilla signing infrastructure.
+This requires that the github repo is cloned to the `github.com/mozilla-extensions` organization, and have a working `.taskcluster.yml`
+
+The Mozilla Taskcluster CI will then automatically start working for this new clone.
+
+NOTE - this must be a clone and not a fork (TODO - file/link to bug on TC side about this?)
+
+## Finding the QA-signed extension
+
+A good CI run will look like:
```

Comment:
how does one get to this url? Through the github actions? Something else?

Reply:
looks like it's a github action: https://github.com/mozilla-extensions/ion-core-addon/runs/1261759243